packages/common/src/webrtc/BaseCall.ts

```diff
@@ -69,7 +69,10 @@ export default abstract class BaseCall implements IWebRTCCall {
     this._registerPeerEvents()
   }
 
-  answer() {
+  answer(params?: { iceTransportPolicy?: RTCConfiguration['iceTransportPolicy'] }) {
+    if (params && params?.iceTransportPolicy) {
+      this.options.iceTransportPolicy = params?.iceTransportPolicy
```

Comment:
We may want to restrict the values that can be assigned to `iceTranportPolicy` to either 'all' (default) or 'relay'.

https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection#icetransportpolicy